vmm/src/memory_manager.rs

```diff
@@ -68,6 +69,12 @@ struct HotPlugState {
     removing: bool,
 }
 
+pub struct Range {
```

Comment:
What is iova?, is that a hva and user_addr a gpa? With such a general name as `Range` I wouldn't expect it to use VFIO specific names. If you wan to have a specific VFIO related type name it as such.

Reply:
Yes good point, I'm gonna go with:
```rust
pub struct Range {
    gpa: u64,
    size: u64,
    hva: u64,
}
```